nx/source/services/capsdc.c

```diff
@@ -57,3 +60,26 @@ Result capsdcShrinkJpeg(u32 width, u32 height, const CapsScreenShotDecodeOption
         }
     );
 }
+
+Result capsdcShrinkJpegEx(u32 scaled_width, u32 scaled_height, u32 jpeg_quality, const CapsScreenShotDecodeOption *opts, const void* jpeg, size_t jpeg_size, void* out_jpeg, size_t out_jpeg_size, u64 *out_result_size) {
+    if (hosversionBefore(18,0,0))
+        return MAKERESULT(Module_Libnx, LibnxError_IncompatSysVer);
+
+    const struct {
+        u32 scaled_width;
+        u32 scaled_height;
+        u32 jpeg_quality;
```

Comment:
Should match the actual IPC param (wiki has u8).

Reply:
Oh I wrote the wiki entry yesterday. I poked deeper and it's u32. I'll correct the wiki entry.